tests/FakeItEasy.Tests/ACallToTests.cs

```diff
@@ -50,5 +68,22 @@ public void CallTo_with_function_call_should_return_configuration_from_configura
             // Assert
             result.Should().BeSameAs(configuration);
         }
+
+        [SuppressMessage("Microsoft.Design", "CA1062:Validate arguments of public methods", MessageId = "0", Justification = "It's not really a public method")]

```

Comment:
@thomaslevesque, You should be able to remove these lines now that #846 has been merged.